### PR TITLE
Making list items dynamic

### PR DESCRIPTION
**Pull request by ChrisBaidoo**

### CODE REVIEW

File: main.js [hunk 1]
```diff
@@ -1,10 +1,62 @@
+function convertToBool(str) {
+  if (['true', 'false'].includes(str)) {
+    return str === 'true';
+  }
+}
+
 function toggleToDo(element){
 
-  var faClassName = element.querySelector(".fa");
+  var faClassName = element.getElementsByClassName("fa")[0];
```
Comment (samdemaeyer) [inline main.js]: Please use `const` instead of var everywhere.

File: main.js [hunk 2]
```diff
@@ -1,10 +1,62 @@
...
+  newIcon.classList.add(iconClass)
+  newIcon.classList.add("fa")
+  var newListTitle = document.createElement("span")
+  newListTitle.innerText = toDoItem.name
+  newListItem.appendChild(newIcon)
+  newListItem.appendChild(newListTitle)
+  document.getElementById(listWrapperId).appendChild(newListItem)
+
+});
+
```
Comment (samdemaeyer) [inline main.js:59]: Let's remove all white spaces in the end .. and remove the comments.

File: styles.css [hunk 3]
```diff
@@ -24,6 +24,10 @@
   border-radius: 2px;
 }
 
+.item .fa{
```
Comment (samdemaeyer) [inline styles.css:35]: Please add a space after `.fa`
```css
.item .fa {
  ...
}
```

**Comment by samdemaeyer** [comment]
@ChrisBaidoo, FYI, I have installed a linter that will tell you certain rules that you should respect. We will discuss it next time.
In the meantime, please make sure you pull my changes before applying any of the comments. (`ggpull`)

File: main.js (hunk 1 above)
Comment (ChrisBaidoo) [reply main.js]: Addressed with this commit; f1a80f1fdaa38cd64bf49d5c6bc33ec3ca4c7c1f

**Pull request merged**